run/test.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import os
+import random
+from datetime import datetime
+
+import torch
+import numpy as np
+import torch.optim as optimizer_modules
+from tqdm import tqdm
+
+from .runnable_utils import merge_configs, init_data_loader
+from utils import DatasetType, Workspace
+from utils import find_cls, load_json, prepare_device
+
+def evaluate(device, prefix, model, data_loader, loss_fn, metric):
+    model.eval()
+
+    total_loss = 0
+    total_metric = 0
+
+    model.train()
+    for _, (data, target) in enumerate(tqdm(data_loader, desc=f"Evaluating {prefix} dataset")):
+        data, target = data.to(device), target.to(device)
+
+        output = model(data)
+        loss = loss_fn(output, target)
+
+        total_loss += loss.item()
+        total_metric += metric(output, target)
+
+    log = {
+        "loss": total_loss / len(data_loader),
+        "metric": total_metric / len(data_loader)
+    }
+
+    return log
+
+def main(config):
+
+    # set random seed
+    torch.manual_seed(config["seed"])
+    np.random.seed(config["seed"])
+    random.seed(config["seed"])
+
+
+    # prepare hardware accelearation
+    device, gpu_device_ids = prepare_device(config["num_gpu"])
+
+    if "cuda" in str(device):
+        print(f"utilizing gpu devices : {gpu_device_ids}")
+        torch.cuda.manual_seed(config["seed"])
```

Comment:
As of PyTorch 1.2, `torch.cuda.manual_seed` isn't needed since `torch.manual_seed` handles it all.

https://pytorch.org/docs/stable/torch.html#torch.manual_seed